chia/wallet/nft_wallet/nft_off_chain.py

```diff
@@ -0,0 +1,96 @@
+import logging
+from pathlib import Path
+from typing import Optional
+
+import requests
```

Comment:
@ytx1991 @altendky I was reading about https://www.python-httpx.org/ recently -- a drop-in replacement for `requests`, but with async support (using a different API). I don't see other uses of requests in the codebase. Is it worth considering this other package?

Reply:
For now we would just use the aiohttp client wouldn't we?  But sure, my impression, without being a major http user until here, is that httpx is a nice and newer lib.  There are options other than aiohttp for serving http as well that we could consider.